data-raw/mdt-dict.csv

```diff
@@ -1,25 +1,25 @@
-"AMDI_0010_I_0001_1","**Falsch.**","**Incorrect.**","**Incorrect.**"
-"AMDI_0011_I_0001_1","Frage **{{num_question}}** von **{{test_length}}**","Question **{{num_question}}** out of **{{test_length}}**","Question **{{num_question}}** de **{{test_length}}**"
-"AMDI_0012_I_0001_1","Die dritte Version wurde verändert.","The third melody was the odd one out.","C'est la troisième mélodie qui était différente."
-"AMDI_0013_I_0001_1","Bei welcher Version wurde ein Ton verändert?","Which melody was the odd one out?","Quelle version était différente?"
-"AMDI_0014_I_0001_1","Du hast den Test beendet!","You finished the test!","Le test est terminé!"
-"AMDI_0015_I_0001_1","Bitte klicke unten auf Beenden.","Please click below to finish.","Veuillez cliquer ci-dessous pour terminer."
-"AMDI_0016_I_0001_1","Weiter","Next","Suivant"
-"AMDI_0017_I_0001_1","Dein Ergebnis war:","Your score was:","Votre score est:"
-"AMDI_0018_I_0001_1","Melodieunterscheidungsquotient (MUQ)","Melodic Discrimination IQ","Niveau Discrimination Mélodique"
-"enter_p_id","Bitte gib Deine Teilnehmer-ID ein.","Please enter your participant ID.","Veuillez noter votre identifiant."
-"results_have_been_saved","Deine Ergebnisse sind gespeichert.","Your results have been saved.","Vos résultats ont été enregistrés."
-"you_may_close_browser","Du kannst dieses Fenster jetzt schließen.","You may now close the browser window.","Vous pouvez maintenant fermer la fenêtre de votre navigateur."
-"demo_intro","Willkommen zum **Melodie-Unterscheidungs-Test**!\\Dieser Test passt sich an Deine Leistung an. Je mehr richtige Antworten Du gibst, desto schwieriger werden die nächsten Stimuli.\\In echten Testanwendungen läuft dieser adaptiver Mechanismus unsichtbar im Hintergrund. Aber in dieser Demoanwendung werden Dir bei jedem Durchgang Deine geschätzte Testfähigkeit (""ability"") und die Schwierigkeit des Stimulus (""difficulty"") angezeigt. ","Welcome to the **Melodic Discrimination Test**!\\This test adapts to your performance. If you do well, you will be given harder items.\\Usually this adaptive procedure is hidden from the participant. For this demonstration, however, we will show your running ability score and corresponding difficulty level on each trial.","Bienvenue au **Test de Discrimination Melodique**!\\Ce test s'adapte à vos performances. Si vos réponses sont correctes, la difficulté du test augmente.\\Normalement, cette procédure n'est pas visible et le participant ne reçoit pas de feedback. Cependant, pour cette démonstration, nous vous montrons votre score au cours du test et le niveau de difficulté des items proposés."
+"ID","DE","DE_F","EN","FR"
```

Comment:
I see you have added German (formal) here, but we originally had German (formal) in dict-german-formal.csv, is this a mistake?

Reply:
In the original mdt-dict DE_F columns are two small mistakes, so dict.german-formal is the correct one, which is now used, as the original column is overwritten. After some log file and commits digging, it looks that I added DE_F to mdt_dict on Nov. 15, 2021, probably because I needed this urgently, afterwards I asked you to add formal German, which you did on Dec 13, 2021,  without noticing that de_f was already in mdt-dict, or something like this. I removed de_f from mdt-dict.csv